src/router/resolvers/index.ts

```diff
@@ -1552,11 +1552,18 @@ export const IGCESupportingDocumentationResolver = (current: string): string =>
 };
 
 export const CurrentlyHasFundingResolver = (current: string): string => {
-  return Summary.hasCurrentStepBeenVisited && current === routeNames.SupportingDocumentation
+  return Summary.hasCurrentStepBeenVisited
     ? routeNames.SummaryStepEight
-    : routeNames.CurrentlyHasFunding
+    : isDitcoUser()
+      ? routeNames.CurrentlyHasFunding
+      : routeNames.RFD
 };
 
+export const GTCInformationResolver = (current: string): string => {
+  return FinancialDetails.hasFunding === "HAS_FUNDING"
+    ? routeNames.GTC
+    : routeNames.GeneratingPackageDocuments
+}

```

Comment:
use `routeNames.routeNames.GeneratingPackageDocumentsFunding` instead.  `routeNames.GeneratingPackageDocuments` navigates the user to the components in step 9 and then if the user clicks back/continue they remain in step 9

Reply:
Updated resolver to use `routeNames.GeneratingPackageDocumentsFunding`